### DISCUSSION TITLE
Introduction about form validation in PrimeVue homepage of V3.47.2 was removed

### DISCUSSION THREAD

**Discussion by DavissonA9** [primefaces/community#1110] [PrimeVue] [answered]
time: 2024-02-05
page: https://primevue.org/autocomplete/#forceselection
version: V3.47.2

description: After PrimeVue homepage was updated to V3.47.2, in the catalog of components/form, every thing about form validation was removed;  
I cannot find any notices about this removement in this issue [issue 5071](https://github.com/primefaces/primevue/issues/5071).
Is there any offical notice about this changes in PrimeVue homepage? 
Why does the form validation is removed?
Why introduces yup and vee-validate in previous version and now remove it in homepage?

**Comment by tugcekucukoglu** [comment, marked as answer]
This decision was made regardless of the issue you mentioned. Improvements in documentation will continue.

Thanks for your understanding!

**Reply by DavissonA9** [reply]
Thanks for your reply, anyway.

And also, are there **any alternatives** to form validation in the future? Or a form validation method integrated into PrimeVue?
